metaflow/metaflow_config.py

```diff
@@ -63,12 +63,12 @@ def from_conf(name, default=None):
 DATATOOLS_SUFFIX = from_conf('METAFLOW_DATATOOLS_SUFFIX', 'data')
```

Comment:
This needs to be sanitized as well

Reply:
Fair point, I suppose with a strip in both directions to be safe there :).

Reply:
Done.  Please review it. Thank You :)